docs/modules/user-guide/partials/proc_stage-1-build-and-package-a-devfile-registry.adoc

```diff
@@ -0,0 +1,174 @@
+[id="stage-1-build-and-package-a-devfile-registry_{context}"]
+== Stage 1: Build and Package a Devfile Registry
+
+The main goal of this stage is:
+
+. Pull in resources into the registry as part of the registry build.
+. Modify the devfile to update references to those offline resources as part of the registry build.
+
+.Prerequisites
+
+* Golang 1.17.x or higher
+* Docker 17.05 or higher or Podman 4.0.x or higher
+* Git
+* Curl
+* Archive Tools (such as `unzip`)
+
+=== Create Offline Registry
+
+Download / clone the link:https://github.com/devfile/registry[devfile/registry] repository.
+
+.Procedure
+
+* Clone through HTTPS
++
+[source,bash]
+----
+git clone https://github.com/devfile/registry.git /path/to/registry
+----
++
+* Clone through SSH
++
+[source,bash]
+----
+git clone git@github.com:devfile/registry.git /path/to/registry
+----
+
+.Additional resources
+
+* To create your own registry Git repository, see xref:building-a-custom-devfile-registry.adoc[Building a custom devfile registry]
+
+=== Packaging Starter Projects
+
+To package starter projects you will need to download them manually then place them under `/stacks/<stack>/<zip>-offline.zip`.
+
+*Note*: Starter projects must be packaged under an archive with the suffix `-offline` to be pulled into the registry.
+
+.Procedure
```

Comment:
I think just including an example downloading the zip and cloning a repo is sufficient here. We don't need the "template" before each one.

Reply:
> I think just including an example downloading the zip and cloning a repo is sufficient here. We don't need the "template" before each one.

Done.